### PR TITLE
[DOCS] tools/docker-compose/README.md: add way to solve postgresql issue

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Relates to https://github.com/ansible/awx/issues/14159

Here's a solution by @fosterseth

I'm not sure if it helps. If you think there's no value in such a note, just please close the PR

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - Docs

##### AWX VERSION

```
devel
```

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @fosterseth

**Review by rooftopcellist — APPROVED** [review]

File: tools/docker-compose/README.md [hunk 1]
```diff
@@ -118,6 +118,12 @@ $ make docker-compose
 
 > For running docker-compose detached mode, start the containers using the following command: `$ make docker-compose COMPOSE_UP_OPTS=-d`
 
+If you have encountered the infinitely-repeating `Waiting for postgres to be ready to accept connections` message during the execution, try to do the following:
```
Comment (masbahnana) [inline tools/docker-compose/README.md:123]: could you link some examples or commands for this section? I think can be helpful for novices.

Comment (Andersson007) [reply tools/docker-compose/README.md:123]: @masbahnana SGTM
- added a general example
- created an additional associated sub-section

thanks, PTAL

**Comment by Andersson007** [comment]
@rooftopcellist @masbahnana thanks for reviewing!
@TheRealHaoLiu thanks for merging!

**Pull request merged**